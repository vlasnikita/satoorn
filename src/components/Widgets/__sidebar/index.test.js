import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import WidgetsSidebar from "./index";

it('should render WidgetsSidebar correctly', () => {
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<WidgetsSidebar />)
    expect(wrapper).toMatchSnapshot()
})