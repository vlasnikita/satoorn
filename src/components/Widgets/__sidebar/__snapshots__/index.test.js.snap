// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render WidgetsSidebar correctly 1`] = `
<div
  className="Widgets__sidebar"
>
  <div
    className="Widgets__sidebarHeader"
  >
    <h1
      className="Widgets__sidebarHeaderText"
    >
      ДОБАВИТЬ ВИДЖЕТ:
    </h1>
  </div>
  <div
    className="Widgets__sidebarList"
  >
    <button
      className="Widgets__sidebarItem"
      onClick={[Function]}
    >
      <div
        className="Widgets__sidebarItemIconContainer"
      >
        <img
          className="Widgets__sidebarItemIcon"
          height={22}
          src="widgets-type-alert.svg"
        />
      </div>
      <p
        className="Widgets__sidebarItemText"
      >
        Алерты и оповещения
      </p>
    </button>
    <button
      className="Widgets__sidebarItem"
      onClick={[Function]}
    >
      <div
        className="Widgets__sidebarItemIconContainer"
      >
        <img
          className="Widgets__sidebarItemIcon"
          height={18}
          src="widgets-type-invite.svg"
        />
      </div>
      <p
        className="Widgets__sidebarItemText"
      >
        Инвайт-виджет
      </p>
    </button>
    <button
      className="Widgets__sidebarItem"
      onClick={[Function]}
    >
      <div
        className="Widgets__sidebarItemIconContainer"
      >
        <img
          className="Widgets__sidebarItemIcon"
          height={22}
          src="widgets-type-challenge.svg"
        />
      </div>
      <p
        className="Widgets__sidebarItemText"
      >
        Челленджи
      </p>
    </button>
  </div>
</div>
`;
